services/wercker/wercker.service.js

```diff
@@ -65,8 +61,8 @@ module.exports = class Wercker extends BaseJsonService {
         result: 'no builds',
       })
     }
-    const { status, result } = json[0]
```

Comment:
Since we're removing `status` from the schema, render fn, etc. can we also remove the the `status` argument passed in the examples?

Reply:
Good spot, I missed some of them!